security/advancedtls/advancedtls.go

```diff
@@ -184,15 +186,26 @@ func (o *ClientOptions) config() (*tls.Config, error) {
 		return nil, fmt.Errorf(
 			"client needs to provide custom verification mechanism if choose to skip default verification")
 	}
+	rootCAs := o.RootCACerts
+	if o.VType < SkipVerification && o.RootCACerts == nil && o.GetRootCAs == nil {
```

Comment:
Using an inequality with an enum is a little unusual.  In this case, I think it would be clearer as `if o.VType != SkipVerification`.

Reply:
Sure! Thanks for pointing that out :)